src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -44,4 +46,9 @@ public LocalSessionFactoryBean getSessionFactory() {
         factoryBean.setPackagesToScan("mate.academy.spring.model");
         return factoryBean;
     }
+
+    @Bean
+    public PasswordEncoder getPasswordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
```

Comment:
Create this bean in SecurityConfig

Reply:
won't move the bean to SecurityConfig class as it's causing exception when running Tomcat